### PR TITLE
Cordova create

### PR DESCRIPTION
**Pull request by DavidThomasSL**
Add Cordova tasks to the Gulp buildfile.

These allow you to do the initial setup of Cordova, and then do the Cordova build.

### CODE REVIEW

**Comment by DavidThomasSL** [comment]
We probably want to copy back the output of the Cordova build when we've finished.

We also need the full build to do a recreate.

File: gulpfile.js [hunk 1]
```diff
@@ -9,10 +9,26 @@ var concatVendor = require('gulp-concat-vendor');
 var streamSeries = require('stream-series');
 var htmlReplace = require('gulp-html-replace');
 var sass = require('gulp-sass');
+var cordova = require('cordova-lib').cordova.raw;
+var del = require('del');
+var fs = require('fs');
```
Comment (tylerferguson) [inline gulpfile.js]: Do we still need fs?

**Comment by ghost** [comment]
Yes, we'll want a desktop build containing HTML, minified JavaScript and CSS, and then another build folder for mobile containing Cordova packages for Android, iOS and Blackberry.

**Comment by DavidThomasSL** [comment]
:ship:

**Pull request merged**